### ISSUE TITLE
OpenTasks widget does not automatically update task list

### ISSUE THREAD

**Issue by hg2581** [dmfs/opentasks#632]
It seems the only way to do this is to delete the widget and create a new one that will include the new task list.

**Comment by dmfs** [comment]
This is intended behavior, new list should not be added automatically. That being said, we should fix #631 to allow adding new lists manually without having to delete the widget.

**Comment by hg2581** [comment]
There could be a settings menu for the widget as well where one option is to automatically add new task lists and/or to automatically use the same tasks lists the app uses (this would be the intuitive behavior). I believe the OpenTasks app automatically adds new lists once CalDav-sync has added them.

**Comment by dmfs** [comment]
Right, a check box "include new task lists" could do the job. We need a way for the widget to figure out which lists are new. Since the row ids are monotonically increasing we could just store the highest row id number and add new lists whenever it finds one with a higher id.

**Comment by hg2581** [comment]
The app itself automatically adds new task lists, does it not? If so, the widget should behave the same.

**Comment by dmfs** [comment]
Ah, so you mean a "show all" option? If we always show everything it's indeed trivial to include new lists (it happens automatically if don't filter by list). If the user selects a subset of all lists, it becomes harder to "detect" new ones.

**Comment by hg2581** [comment]
Yes. Again I use Business Tasks 2 as an example, their widget has options to use the same lists as the app itself, or choose a subset. Suggest the same for OpenTask widget.

**Comment by boggle247** [comment]
I have this issue too. I've deleted the widget and even uninstalled the app. The widget never updates with any tasks. Whether I create a task from the widget or from within the app, doesn't work.
